packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -698,10 +780,13 @@ const Conversation = WebexPlugin.extend({
    * @returns {Promise}
    */
   unassign(conversation, activity) {
-    return this._inferConversationUrl(conversation)
+    const url = this.getConvoUrl(conversation);
```

Comment:
These two lines are repeated several times. It may make sense to encapsulate them in a method.